test/libsolidity/ViewPureChecker.cpp

```diff
@@ -92,6 +92,26 @@ BOOST_AUTO_TEST_CASE(environment_access)
 		TypeError,
 		"\"block.blockhash()\" has been deprecated in favor of \"blockhash()\""
 	);
+
+	std::string baseFeeContract = "contract C { function f() view public { block.basefee; } }";
```

Comment:
Oh! Shouldn't we do that in another PR for `basefee`? I just noticed that it is also missing `prevrandao` there, so it would be nice to create another PR adding all missing global variables and functions.

Reply:
`prevrandao` is technically covered, since `block.difficulty` is in there - which has the same opcode as `prevrandao`, which means it falls into the same case (in the instruction switch), and then makes a check based on the EVM version as to which one to use.

Reply:
Also, the changes related to `basefee` were small enough that IMO it's fine to put them here. Would not hurt to have them in a separate commit, but that's just a small detail.